lib/twine/formatters/android.rb

```diff
@@ -45,7 +45,7 @@ def output_path_for_language(lang)
       end
 
       def set_translation_for_key(key, lang, value)
-        value = CGI.unescapeHTML(value)
+        value = CGI.unescapeHTML((value.nil? || value.empty?) ? "" : value)
```

Comment:
What is the purpose behind this change? Was this a bug in the old code? And if so how did it occur?